grpc/channelz/v1/channelz.proto

```diff
@@ -437,12 +438,21 @@ service Channelz {
 message GetTopChannelsRequest {
   // start_channel_id indicates that only channels at or above this id should be
   // included in the results.
+  // To request the first page, this must be set to 0. To request
```

Comment:
s/must/should/   Setting it to 1 should have identical results, since the bounds are inclusive

Reply:
You're right.